src/blocks/music.rs

```diff
@@ -24,8 +24,12 @@
 //! `interface_name_exclude` | A list of regex patterns for player MPRIS interface names to ignore. | `["playerctld"]`
 //! `separator` | String to insert between artist and title. | `" - "`
 //! `seek_step_secs` | Positive number of seconds to seek forward/backward when scrolling on the bar. Does not need to be an integer. | `1`
+//! `seek_forward_step_secs` | Positive number of seconds to seek forward when scrolling on the bar. Does not need to be an integer. | `1`
+//! `seek_backward_step_secs` | Positive number of seconds to seek backward when scrolling on the bar. Does not need to be an integer. | `1`
 //! `volume_step` | The percent volume level is increased/decreased for the selected audio device when scrolling. Capped automatically at 50. | `5`
 //!
+//! Note: `seek_forward_step_secs` and `seek_backward_step_secs` are mutually exclusive with `seek_step_secs`.
```

Comment:
Maybe instead we can treat `seek_step_secs` as a default value for `seek_{forward,backward}_step_secs`? I'm just not sure if added validation is worth it.

Reply:
Sounds good to me, I can change that.